function/random/random.js

```diff
@@ -4,49 +4,63 @@ module.exports = function(RED) {
     function RandomNode(n) {
         RED.nodes.createNode(this,n);
 
-        this.low  = Number(n.low) || ""; 
-        this.high = Number(n.high) || "";
+        this.status({}); // blank status on a deploy
+
         this.inte = n.inte || false;
         this.property = n.property||"payload";
         var node = this;
-        
-      
+
+
         this.on("input", function(msg) {
+ 
+           if (n.low !== "") {this.low  = parseFloat(n.low)}
+              else {this.low = n.low}; 
+           if (n.high !== "") {this.high  = parseFloat(n.high)}
+              else {this.high = n.high}; 
+      
+           if (node.low === "") {
+               node.low = 1;
+               if ('from' in msg) {node.low = msg.from} 
```

Comment:
we can't just change node.low as that would then become a permanant change next time it was called. must use a a new variable for this.

Reply:
This was fixed by using a new variable